samples/ClassroomLabs/Modules/Library/Az.LabServices.psm1

```diff
@@ -1297,6 +1298,46 @@ function InvokeStudentRest {
     Write-Verbose $body
     return Invoke-RestMethod -Uri $fullUri -Method 'Post' -Headers $headers -Body $body -ContentType 'application/json'
 }
+
+function Add-AzLabStudentUsage {

```

Comment:
I think I am getting confused with this one. Why this functionality is not implemented on Set-AzLabUser?